modules/metering-uninstall.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-uninstall.adoc
+
+[id="metering-uninstall_{context}"]
+= Uninstalling metering from {product-title}
+
+You can remove metering from your cluster.
+
+.Prerequisites
+
+* Metering must be installed.
+
+.Procedure
+
+To remove metering:
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Find the metering operator and click the {kebab} menu. Select *Uninstall Operator*.
+
+. A dialog opens. Click *Remove* to uninstall metering.
+
+[NOTE]
+====
```

Comment:
@lamek this section is duplicated in `modules/metering-store-data-in-s3.adoc` ...unless that's intended.

Reply:
I left that intentionally. I think the behavior is unintuitive and therefore good to call out where customers are most likely to see it. Unless your strongly opposed, I vote we keep this in both locations.

Reply:
@lamek ok that's fine....the rest lgtm